packages/ui/src/icons/index.tsx

```diff
@@ -1 +1,223 @@
-export * from './chevron';
+import React from 'react';
```

Comment:
This file will continue to grow over time — it's already too big imo. It'll be cleaner to give each icon its own file, and export them from the index. What are your thoughts?

Reply:
It also means we can easily jump to the file of a given icon, as it has a descriptive name — not just `index.tsx`.

Reply:
It also means we can easily jump to the file of a given icon, as it has a descriptive name — not just `index.tsx`.

Reply:
Yep agreed--I'll make them their own files